README.md

```diff
@@ -67,12 +67,12 @@ Asserts that the target is False:
 expect(False).to.be.false()
 ```
 
-#### instanceof
+#### instance_of
```

Comment:
I don't think it's a good reason for changing this, a small change in your lib make people have a lot of time to fix this in their code. I think it's better to revert this and we can consider it later in some major release.